src/Type/Object/ThemeMods.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace WPGraphQL\Type;
+
+use WPGraphQL\Data\DataSource;
+
+register_graphql_object_type( 'ThemeMods', [
+	'description' => __( 'All of registered theme modifications' ),
+	'interfaces'  => [ WPObjectType::node_interface() ],
+	'fields'  => [
+		'background' => [ 
+			'type' 			=> 'MediaItem',
+			'description'	=> __( 'custom background' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {	
+				if( ! empty( $root['background'] ) ) { 
+					return ( ! empty( $root['background']['id'] ) ) ?
+					DataSource::resolve_post_object( absint( $root['background']['id'] ), 'attachment' ) :
+					null;
+				}
+
+				return null;
+			}
+		],
+		'backgroundColor' => [ 
+			'type' 			=> 'String',
+			'description'	=> __( 'background color' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {
+				return ( ! empty( $root['background_color'] ) ) ? $root['background_color'] : null;
+			}
+		],
+		'customCssPostId' => [ 
+			'type' 			=> 'Int',
+			'description'	=> __( 'custom theme logo' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {
+				return ( ! empty( $root['custom_css_post_id'] ) ) ? $root['custom_css_post_id'] : null;
+			}
+		],
+		'customLogo' => [ 
```

Comment:
This should maybe be a more complex type than `Int`. . .like a `MediaItem` perhaps?

Reply:
Hmm. If we're going to expose the PostObjects for the `custom_css` [post_type](https://codex.wordpress.org/Post_Types#Default_Post_Types), we probably should register the `custom_css` as `show_in_graphql => true` and give it a `graphql_single_name` and `graphql_plural_name`. 

Then, instead of resolving to `type => Post` we can resolve to `type => CustomCSS` or something to that tune. 

And the resolver should also be changed to `DataSource::resolve_post_object( absint( $root['custom_css_post_id'] ), 'custom_css' )`

Right now it's misleading that CSS is a "post" when really it's not, especially when the Schema is filtered by various plugins to shape the Schema more for their particular applications. 

The CustomCSS PostObject Type should have the ability to be shaped differently than the `Post` type in the Schema.

Reply:
I agree, but I thought it should have a separate issue and branch like the handling of the `custom-background` and `custom-header` context data issue I mentioned above.

Reply:
🤔 

I don't want to intentionally add this to resolve to `Post` if we _know_ it's for sure going to be a breaking change to make the switch to resolve to `CustomCSS` later.

Reply:
so return it to an `ID` until the `customCSS` `post-object` is created?

Reply:
🤔 

Hmm. . .

I dunno the best approach here. 

If we expose the field `customCssPostId` and just return the ID, I suppose that might be ok for now, then come back and add a `customCSS` field that resolves as an object of the `CustomCSS` type that could work.